Basic/Pod/QuickStart.pod

```diff
@@ -41,17 +41,15 @@ data to be easily played with and visualised.
 
 =head2 Help
 
-PDL contains extensive documentation, available both within the
+PDL contains extensive user documentation, available both within the
```

Comment:
Probably also worth mentioning MetaCPAN these days! I use `perldl`'s `help (or ?)` function, sometimes `perldoc`, and the web. I've never used `pdldoc`.

Reply:
I will add a link to MetaCPAN.  I'll also add a line describing `pdldoc` as the equivalent of `perldoc -f` which in my opinion makes a lot of sense.